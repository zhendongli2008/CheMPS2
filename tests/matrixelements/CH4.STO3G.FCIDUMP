 &FCI NORB= 9,NELEC= 10,MS2= 0,
  ORBSYM=1,1,1,1,1,2,2,3,3,
  ISYM=1,
 /
  3.5013086635273840E+00   1   1   1   1
  2.9301788627138348E-01   2   1   1   1
  3.8510801741227410E-02   2   1   2   1
  7.0713104137852856E-01   2   2   1   1
  9.1536054235476887E-03   2   2   2   1
  5.0257408070517517E-01   2   2   2   2
 -1.9266479977367626E-11   3   1   1   1
 -1.7758101846648833E-12   3   1   2   1
 -3.1159204990499623E-12   3   1   2   2
  8.7227321460576109E-03   3   1   3   1
  3.2530111654242074E-12   3   2   1   1
 -1.4921251083668818E-12   3   2   2   1
  2.1912239384080934E-11   3   2   2   2
 -1.3677730408308135E-02   3   2   3   1
  1.0775632714855414E-01   3   2   3   2
  6.3103799121431237E-01   3   3   1   1
  3.8877435079642669E-03   3   3   2   1
  4.8366395295058984E-01   3   3   2   2
  1.3573760171414762E-12   3   3   3   1
 -9.3429985970061580E-12   3   3   3   2
  4.8842070134961113E-01   3   3   3   3
 -7.7340874356125603E-10   4   1   1   1
 -1.0232352373176420E-10   4   1   2   1
 -2.6493548891184937E-11   4   1   2   2
  1.3371913217734772E-02   4   1   3   1
 -1.8978256365317865E-02   4   1   3   2
 -6.6117450056579763E-12   4   1   3   3
  2.0623742415583353E-02   4   1   4   1
 -6.6636476900957706E-10   4   2   1   1
 -2.4914891601657968E-11   4   2   2   1
 -2.0070196470479279E-10   4   2   2   2
 -9.8222009631055776E-03   4   2   3   1
  1.8805965860461363E-02   4   2   3   2
 -1.6064392069185807E-10   4   2   3   3
 -1.4279615920789756E-02   4   2   4   1
  4.7728040555207379E-02   4   2   4   2
  2.5099511136724767E-01   4   3   1   1
  5.4187836395156869E-03   4   3   2   1
  8.1297369993863222E-02   4   3   2   2
  1.6685132442351147E-11   4   3   3   1
  2.4358078054564913E-11   4   3   3   2
  7.7409782898452864E-02   4   3   3   3
  1.2344288785604007E-11   4   3   4   1
 -2.7014814690806688E-10   4   3   4   2
  8.6448250541825644E-02   4   3   4   3
  7.2981409414396081E-01   4   4   1   1
  8.4116166043502784E-03   4   4   2   1
  4.9207921197177118E-01   4   4   2   2
  6.6230257389499059E-11   4   4   3   1
 -4.5759154798652446E-10   4   4   3   2
  4.8974017141301490E-01   4   4   3   3
  7.6875047994029444E-11   4   4   4   1
 -2.3277280103428666E-10   4   4   4   2
  8.8218575691677459E-02   4   4   4   3
  5.1380712421383079E-01   4   4   4   4
  3.2572401042219545E-01   5   1   1   1
  4.3608805220476388E-02   5   1   2   1
  9.6415652277579058E-03   5   1   2   2
  2.9692521878732503E-11   5   1   3   1
 -4.6347667945288024E-11   5   1   3   2
  3.7101422024034855E-03   5   1   3   3
 -6.7114046958804963E-11   5   1   4   1
 -6.0006957961424739E-11   5   1   4   2
  5.2281557611224203E-03   5   1   4   3
  8.0327569072608396E-03   5   1   4   4
  4.9451150306469571E-02   5   1   5   1
  2.8744661347814021E-01   5   2   1   1
  1.0323551868956730E-02   5   2   2   1
  8.7807085842504212E-02   5   2   2   2
 -2.4612782298372160E-11   5   2   3   1
  4.2588235021900900E-11   5   2   3   2
  6.8530137305224448E-02   5   2   3   3
 -6.0696850323616047E-11   5   2   4   1
 -5.3517100473410295E-11   5   2   4   2
  7.1906736206095001E-02   5   2   4   3
  8.1788028642568247E-02   5   2   4   4
  1.0843815388236047E-02   5   2   5   1
  7.3555165980477594E-02   5   2   5   2
  5.7885896688705959E-10   5   3   1   1
  1.1872392636060394E-11   5   3   2   1
  1.8371069249178973E-10   5   3   2   2
 -6.7250625674702656E-03   5   3   3   1
 -1.0777119677991488E-02   5   3   3   2
  1.7950846711856094E-10   5   3   3   3
 -9.8106283663600396E-03   5   3   4   1
  4.1411265060605718E-02   5   3   4   2
  8.3318085070116510E-11   5   3   4   3
  2.9479684665600914E-10   5   3   4   4
 -1.1900641713399884E-11   5   3   5   1
  2.6861029372993173E-10   5   3   5   2
  4.7236783951143245E-02   5   3   5   3
  1.1100752021970303E-10   5   4   1   1
 -9.5479587777780317E-12   5   4   2   1
  7.9291373814016630E-11   5   4   2   2
 -1.3984961907857592E-02   5   4   3   1
  9.6308681223180551E-02   5   4   3   2
  6.8473088310483377E-11   5   4   3   3
 -2.0111453311237079E-02   5   4   4   1
  9.1147266199284593E-03   5   4   4   2
  1.2278260352882597E-10   5   4   4   3
 -3.9115188865679329E-10   5   4   4   4
 -5.9102565289764764E-11   5   4   5   1
  6.2090797781078066E-11   5   4   5   2
 -1.9073335053460969E-02   5   4   5   3
  1.0174343065748673E-01   5   4   5   4
  6.9102246308565762E-01   5   5   1   1
  1.2119673740093696E-02   5   5   2   1
  4.6842446936260518E-01   5   5   2   2
 -6.8373771922036752E-11   5   5   3   1
  4.6630874162012503E-10   5   5   3   2
  4.5775649172038335E-01   5   5   3   3
 -1.2714821556736311E-10   5   5   4   1
 -1.0674544581590339E-10   5   5   4   2
  5.6669441698313616E-02   5   5   4   3
  4.7588749010857340E-01   5   5   4   4
  1.2577559946693080E-02   5   5   5   1
  6.5263500655151852E-02   5   5   5   2
  3.8500619359282950E-11   5   5   5   3
  5.3263317367768792E-10   5   5   5   4
  4.5979386094116975E-01   5   5   5   5
  6.3103799120174064E-01   6   6   1   1
  3.8877435081782260E-03   6   6   2   1
  4.8366395293492748E-01   6   6   2   2
  3.9336224502351941E-03   6   6   3   1
 -5.8918737556678702E-02   6   6   3   2
  4.6236590243650733E-01   6   6   3   3
  4.7794339903757602E-03   6   6   4   1
  2.0444789977243583E-02   6   6   4   2
  5.5019299042168512E-02   6   6   4   3
  4.7166737715872453E-01   6   6   4   4
  3.7101422138118220E-03   6   6   5   1
  6.8530137354454401E-02   6   6   5   2
  3.1410007940316786E-02   6   6   5   3
 -5.6564875627144198E-02   6   6   5   4
  4.5775649144470643E-01   6   6   5   5
  5.3533314635308982E-01   6   6   6   6
  2.5099511137214531E-01   7   6   1   1
  5.4187836391087693E-03   7   6   2   1
  8.1297370007347686E-02   7   6   2   2
 -3.5244672770480007E-03   7   6   3   1
  5.3204843934557135E-02   7   6   3   2
  5.5019299118568912E-02   7   6   3   3
 -5.3081287090162022E-03   7   6   4   1
 -6.3483705083033559E-03   7   6   4   2
  6.4788196197965595E-02   7   6   4   3
  6.7744933308860888E-02   7   6   4   4
  5.2281557481678419E-03   7   6   5   1
  7.1906736189579462E-02   7   6   5   2
 -2.6120707491024860E-02   7   6   5   3
  5.8037410488009707E-02   7   6   5   4
  5.6669441980342350E-02   7   6   5   5
  3.4443006865736307E-02   7   6   6   6
  1.2104617385785141E-01   7   6   7   6
  7.2981409413870157E-01   7   7   1   1
  8.4116166042315713E-03   7   7   2   1
  4.9207921196325155E-01   7   7   2   2
 -3.7575224667417417E-04   7   7   3   1
 -3.3638696314735594E-02   7   7   3   2
  4.7166737689439753E-01   7   7   3   3
 -1.1944026562951612E-03   7   7   4   1
  3.1432462212674317E-02   7   7   4   2
  6.7744933484618380E-02   7   7   4   3
  4.9050539115820235E-01   7   7   4   4
  8.0327569041922439E-03   7   7   5   1
  8.1788028720545192E-02   7   7   5   2
  3.9573201841670180E-02   7   7   5   3
 -3.9533760561377182E-02   7   7   5   4
  4.7588748991723584E-01   7   7   5   5
  5.2868234268599357E-01   7   7   6   6
  5.3197479592696828E-02   7   7   7   6
  5.5392015802881045E-01   7   7   7   7
  6.3103799122688242E-01   8   8   1   1
  3.8877435077501379E-03   8   8   2   1
  4.8366395296625164E-01   8   8   2   2
 -3.9336224513171585E-03   8   8   3   1
  5.8918737562747764E-02   8   8   3   2
  4.6236590242922754E-01   8   8   3   3
 -4.7794340089952550E-03   8   8   4   1
 -2.0444790295645443E-02   8   8   4   2
  5.5019299201285066E-02   8   8   4   3
  4.7166737662623992E-01   8   8   4   4
  3.7101421909949833E-03   8   8   5   1
  6.8530137255994550E-02   8   8   5   2
 -3.1410007690220926E-02   8   8   5   3
  5.6564875704969222E-02   8   8   5   4
  4.5775649199606050E-01   8   8   5   5
  4.1545345741871004E-01   8   8   6   6
  9.7986075161555108E-02   8   8   7   6
  4.3272520561480243E-01   8   8   7   7
  5.3533314637444551E-01   8   8   8   8
  2.5099511136235053E-01   9   8   1   1
  5.4187836399226270E-03   9   8   2   1
  8.1297369980379258E-02   9   8   2   2
  3.5244672773362372E-03   9   8   3   1
 -5.3204843941666483E-02   9   8   3   2
  5.5019299124885443E-02   9   8   3   3
  5.3081286852672653E-03   9   8   4   1
  6.3483701712038834E-03   9   8   4   2
  6.4788196069432702E-02   9   8   4   3
  6.7744933857866244E-02   9   8   4   4
  5.2281557740768148E-03   9   8   5   1
  7.1906736222611192E-02   9   8   5   2
  2.6120707787315171E-02   9   8   5   3
 -5.8037410432776916E-02   9   8   5   4
  5.6669441416284959E-02   9   8   5   5
  9.7986075150308022E-02   9   8   6   6
  3.0190272817950670E-02   9   8   7   6
  1.0276602968093730E-01   9   8   7   7
  3.4443006862760535E-02   9   8   8   8
  1.2104617385729771E-01   9   8   9   8
  7.2981409414922216E-01   9   9   1   1
  8.4116166044690549E-03   9   9   2   1
  4.9207921198029153E-01   9   9   2   2
  3.7575224566985602E-04   9   9   3   1
  3.3638696321725364E-02   9   9   3   2
  4.7166737689056837E-01   9   9   3   3
  1.1944026180083570E-03   9   9   4   1
 -3.1432462594181865E-02   9   9   4   2
  6.7744933682109598E-02   9   9   4   3
  4.9050539078416916E-01   9   9   4   4
  8.0327569103295707E-03   9   9   5   1
  8.1788028564591830E-02   9   9   5   2
 -3.9573201534813474E-02   9   9   5   3
  3.9533760644297700E-02   9   9   5   4
  4.7588749029991240E-01   9   9   5   5
  4.3272520560751315E-01   9   9   6   6
  1.0276602969060677E-01   9   9   7   6
  4.5039235714983361E-01   9   9   7   7
  5.2868234270268599E-01   9   9   8   8
  5.3197479585843810E-02   9   9   9   8
  5.5392015804155925E-01   9   9   9   9
  1.3027399458371634E-02   8   6   8   6
  1.1195241888834728E-02   8   7   8   6
  1.2405240137689675E-02   8   7   8   7
  1.1195241887891070E-02   9   6   8   6
  9.2548142710929503E-03   9   6   8   7
  1.2405240136377910E-02   9   6   9   6
  9.0363972602661800E-03   9   7   8   6
  1.0236821054230294E-02   9   7   8   7
  1.0236821054083588E-02   9   7   9   6
  1.1650866621322763E-02   9   7   9   7
  8.7227321457937854E-03   6   1   6   1
 -1.3677730407576737E-02   6   2   6   1
  1.0775632713870933E-01   6   2   6   2
  3.9336224516138699E-03   6   3   6   1
 -5.8918737563662366E-02   6   3   6   2
  5.9939844482371316E-02   6   3   6   3
 -3.5244672611636392E-03   6   4   6   1
  5.3204843962654041E-02   6   4   6   2
 -3.1771534226664941E-02   6   4   6   3
  4.7003163579552282E-02   6   4   6   4
 -6.7250625758407219E-03   6   5   6   1
 -1.0777119549090941E-02   6   5   6   2
  3.1410007741032697E-02   6   5   6   3
 -2.6120707640950386E-02   6   5   6   4
  4.7236783827333290E-02   6   5   6   5
  1.3371913217466733E-02   7   1   6   1
 -1.8978256364728417E-02   7   1   6   2
  4.7794340010829947E-03   7   1   6   3
 -5.3081286738276053E-03   7   1   6   4
 -9.8106283790446031E-03   7   1   6   5
  2.0623742415354720E-02   7   1   7   1
 -9.8222009635342868E-03   7   2   6   1
  1.8805965867846632E-02   7   2   6   2
  2.0444790131417118E-02   7   2   6   3
 -6.3483704353546446E-03   7   2   6   4
  4.1411265043722535E-02   7   2   6   5
 -1.4279615921531168E-02   7   2   7   1
  4.7728040554858957E-02   7   2   7   2
 -3.5244672761166680E-03   7   3   6   1
  5.3204843932870630E-02   7   3   6   2
 -3.1771534150982522E-02   7   3   6   3
  4.3852737650649051E-02   7   3   6   4
 -2.6120707538423854E-02   7   3   6   5
 -5.3081286956924746E-03   7   3   7   1
 -6.3483703395871491E-03   7   3   7   2
  4.7003163453339686E-02   7   3   7   3
 -3.7575221282135714E-04   7   4   6   1
 -3.3638696545646093E-02   7   4   6   2
  4.7978568680491974E-02   7   4   6   3
 -2.4784275283092967E-02   7   4   6   4
  3.9573201675624156E-02   7   4   6   5
 -1.1944025890573372E-03   7   4   7   1
  3.1432462379166172E-02   7   4   7   2
 -2.4784275189776953E-02   7   4   7   3
  5.1763900632873359E-02   7   4   7   4
 -1.3984961908636361E-02   7   5   6   1
  9.6308681138755237E-02   7   5   6   2
 -5.6564875559689767E-02   7   5   6   3
  5.8037410448359278E-02   7   5   6   4
 -1.9073334820492234E-02   7   5   6   5
 -2.0111453314279267E-02   7   5   7   1
  9.1147267013831218E-03   7   5   7   2
  5.8037410399582004E-02   7   5   7   3
 -3.9533760723230948E-02   7   5   7   4
  1.0174343046974318E-01   7   5   7   5
  8.7227321463213634E-03   8   1   8   1
 -1.3677730409039437E-02   8   2   8   1
  1.0775632715839877E-01   8   2   8   2
 -3.9336224499384584E-03   8   3   8   1
  5.8918737555764031E-02   8   3   8   2
  5.9939844462686138E-02   8   3   8   3
  3.5244672932205909E-03   8   4   8   1
 -5.3204843913569763E-02   8   4   8   2
 -3.1771534065018293E-02   8   4   8   3
  4.7003163326012599E-02   8   4   8   4
 -6.7250625590997565E-03   8   5   8   1
 -1.0777119806892485E-02   8   5   8   2
 -3.1410007889505119E-02   8   5   8   3
  2.6120707637389960E-02   8   5   8   4
  4.7236784074953408E-02   8   5   8   5
  1.3371913218002778E-02   9   1   8   1
 -1.8978256365907345E-02   9   1   8   2
 -4.7794339982880230E-03   9   1   8   3
  5.3081287204559107E-03   9   1   8   4
 -9.8106283536754189E-03   9   1   8   5
  2.0623742415812128E-02   9   1   9   1
 -9.8222009626768858E-03   9   2   8   1
  1.8805965853076412E-02   9   2   8   2
 -2.0444790141471710E-02   9   2   8   3
  6.3483702441523579E-03   9   2   8   4
  4.1411265077488879E-02   9   2   8   5
 -1.4279615920048469E-02   9   2   9   1
  4.7728040555556059E-02   9   2   9   2
  3.5244672782675525E-03   9   3   8   1
 -5.3204843943352870E-02   9   3   8   2
 -3.1771534140700344E-02   9   3   8   3
  4.3852737523033931E-02   9   3   8   4
  2.6120707739916212E-02   9   3   8   5
  5.3081286985910310E-03   9   3   9   1
  6.3483703399197025E-03   9   3   9   2
  4.7003163452224855E-02   9   3   9   3
  3.7575227952266042E-04   9   4   8   1
  3.3638696090815198E-02   9   4   8   2
  4.7978568402690228E-02   9   4   8   3
 -2.4784274813409314E-02   9   4   8   4
 -3.9573201700859637E-02   9   4   8   5
  1.1944026852461610E-03   9   4   9   1
 -3.1432462427689836E-02   9   4   9   2
 -2.4784274906725176E-02   9   4   9   3
  5.1763900252477110E-02   9   4   9   4
 -1.3984961907078781E-02   9   5   8   1
  9.6308681307605892E-02   9   5   8   2
  5.6564875772423168E-02   9   5   8   3
 -5.8037410472427442E-02   9   5   8   4
 -1.9073335286429832E-02   9   5   8   5
 -2.0111453308195110E-02   9   5   9   1
  9.1147265384743797E-03   9   5   9   2
 -5.8037410521204660E-02   9   5   9   3
  3.9533760482443746E-02   9   5   9   4
  1.0174343084523108E-01   9   5   9   5
 -1.9668337340950966E+01   1   1   0   0
 -3.6653114396764719E-01   2   1   0   0
 -5.3604207365203527E+00   2   2   0   0
  2.6486146609272510E-11   3   1   0   0
 -4.0885739238660790E-11   3   2   0   0
 -4.8447657229839862E+00   3   3   0   0
  9.0068508207252762E-10   4   1   0   0
  2.4626416461615008E-09   4   2   0   0
 -9.9343713132107547E-01   4   3   0   0
 -4.3568878724435827E+00   4   4   0   0
 -3.7711962992559012E-01   5   1   0   0
 -1.0626036904436358E+00   5   2   0   0
 -2.2810300315256882E-09   5   3   0   0
 -7.1406935919782200E-10   5   4   0   0
 -4.0651113690335334E+00   5   5   0   0
 -4.8447657229142065E+00   6   6   0   0
 -9.9343713136181133E-01   7   6   0   0
 -4.3568878724476541E+00   7   7   0   0
 -4.8447657230537597E+00   8   8   0   0
 -9.9343713128034583E-01   9   8   0   0
 -4.3568878724395210E+00   9   9   0   0
  1.3313250311190094E+01   0   0   0   0