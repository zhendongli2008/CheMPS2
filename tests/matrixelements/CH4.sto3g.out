    -----------------------------------------------------------------------
          PSI4: An Open-Source Ab Initio Electronic Structure Package
                              PSI 4.0 Driver

               Git: Rev {master} 93a5e8d84afd652e6226b966a114479f5ca1a459

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: 03-28-2015  10:18:58 PM EDT

    Process ID:  12101
    PSI4DATADIR: /home/seba/hackathon/psi4public/lib
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
molecule CH4 {
   C   0.0                      0.0                      0.0
   H   0.0                      0.0                      2.078698587
   H   1.9598158229645684       0.0                     -0.692899529
   H  -0.9799079114822842       1.6972502894260224      -0.692899529
   H  -0.9799079114822842      -1.6972502894260224      -0.692899529
units au
}

plugin_load("fcidump/fcidump.so")

set basis sto-3g
set reference rhf
set e_convergence 1e-12
set d_convergence 1e-12
set ints_tolerance 0.0

set fcidump dumpfilename CH4.sto3g.fcidump

scf()

plugin("fcidump.so")

--------------------------------------------------------------------------
fcidump/fcidump.so loaded.

*** tstart() called on seb-latitude
*** at Sat Mar 28 22:18:58 2015

    There are an even number of electrons - assuming singlet.
    Specify the multiplicity with the MULTP option in the
    input if this is incorrect


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,    256 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2v
    Full point group: Td

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z       
    ------------   -----------------  -----------------  -----------------
           C          0.000000000000    -0.000000000000     0.000000000000
           H          0.000000000000    -1.697250289185    -1.200137188939
           H          1.697250289426     0.000000000000     1.200137188939
           H         -1.697250289426    -0.000000000000     1.200137188939
           H         -0.000000000000     1.697250289185    -1.200137188939

  Running in c2v symmetry.

  Rotational constants: A =      5.18391  B =      5.18391  C =      5.18391 [cm^-1]
  Rotational constants: A = 155409.66304  B = 155409.66303  C = 155409.66302 [MHz]
  Nuclear repulsion =   13.313250311190094

  Charge       = 0
  Multiplicity = 1
  Electrons    = 10
  Nalpha       = 5
  Nbeta        = 5

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-12
  Density threshold  = 1.00e-12
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: STO-3G
    Number of shells: 7
    Number of basis function: 9
    Number of Cartesian functions: 9
    Spherical Harmonics?: true
    Max angular momentum: 1

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A1         5       5       0       0       0       0
     A2         0       0       0       0       0       0
     B1         2       2       0       0       0       0
     B2         2       2       0       0       0       0
   -------------------------------------------------------
    Total       9       9       5       5       5       0
   -------------------------------------------------------

  Starting with a DF guess...

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):               183
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          0E+00
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 42
    Number of basis function: 112
    Number of Cartesian functions: 126
    Spherical Harmonics?: true
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 2.2657035852E-01.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:   -36.07089275503217   -3.60709e+01   2.60160e-01 
   @DF-RHF iter   2:   -39.54188534143977   -3.47099e+00   9.62443e-02 DIIS
   @DF-RHF iter   3:   -39.72473517306828   -1.82850e-01   9.07248e-03 DIIS
   @DF-RHF iter   4:   -39.72623843718238   -1.50326e-03   9.57876e-04 DIIS
   @DF-RHF iter   5:   -39.72625380656368   -1.53694e-05   1.37614e-04 DIIS
   @DF-RHF iter   6:   -39.72625419326453   -3.86701e-07   1.56902e-06 DIIS
   @DF-RHF iter   7:   -39.72625419330885   -4.43237e-11   2.45345e-08 DIIS
   @DF-RHF iter   8:   -39.72625419330883    2.13163e-14   2.87527e-10 DIIS
   @DF-RHF iter   9:   -39.72625419330887   -3.55271e-14   2.76188e-12 DIIS
   @DF-RHF iter  10:   -39.72625419330885    1.42109e-14   2.48724e-14 DIIS

  DF guess converged.

  ==> Integral Setup <==

 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of atoms:                   5
      Number of AO shells:               7
      Number of SO shells:               5
      Number of primitives:             21
      Number of atomic orbitals:         9
      Number of basis functions:         9

      Number of irreps:                  4
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [   5    0    2    2 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 335 non-zero two-electron integrals.
        Stored in file 33.

	Batch   1 pq = [       0,      21] index = [             0,231]
  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory (MB):               183
    Schwarz Cutoff:          0E+00

   @RHF iter  11:   -39.72591203381210    3.42159e-04   1.74756e-05 DIIS
   @RHF iter  12:   -39.72591203476734   -9.55239e-10   8.76927e-07 DIIS
   @RHF iter  13:   -39.72591203477285   -5.51381e-12   7.55835e-08 DIIS
   @RHF iter  14:   -39.72591203477302   -1.63425e-13   8.55106e-09 DIIS
   @RHF iter  15:   -39.72591203477302   -7.10543e-15   2.01799e-14 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A1   -11.031218     2A1    -0.903381     1B2    -0.513836  
       3A1    -0.513836     1B1    -0.513836  

    Virtual:                                                              

       2B1     0.704242     4A1     0.704242     2B2     0.704242  
       5A1     0.735605  

    Final Occupation by Irrep:
             A1    A2    B1    B2 
    DOCC [     3,    0,    1,    1 ]

  Energy converged.

  @RHF Final Energy:   -39.72591203477302

   => Energetics <=

    Nuclear Repulsion Energy =             13.3132503111900942
    One-Electron Energy =                 -79.1261104928470331
    Two-Electron Energy =                  26.0869481468839055
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                        -39.7259120347730388

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -0.0000      Y:     0.0000      Z:     0.0000

  Electronic Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:    -0.0000

  Dipole Moment: (a.u.)
     X:    -0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000

  Dipole Moment: (Debye)
     X:    -0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000


  Saving occupied orbitals to File 180.

*** tstop() called on seb-latitude at Sat Mar 28 22:18:59 2015
Module time:
	user time   =       0.51 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       0.51 seconds =       0.01 minutes
	system time =       0.03 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Reading options from the FCIDUMP block
Calling plugin fcidump.so.
	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Transforming the one-electron integrals and constructing Fock matrices
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.
Created the file CH4.STO3G.FCIDUMP
*** PSI4 exiting successfully. Buy a developer a beer!
